root/robots.txt.production

```diff
@@ -5,12 +5,14 @@ Disallow: /ajax/
 Disallow: /area/
 Disallow: /cdstub/
 Disallow: /cdtoc/attach
+Disallow: /collection/create
```

Comment:
This will be covered by https://github.com/metabrainz/musicbrainz-server/pull/515/files#diff-d18cb9e7c5f98fd00839f1bfa79034e9R8